src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderStatic.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data.StaticPipe.Runtime;
+using Microsoft.ML.Data.StaticPipe;
+using Microsoft.ML.Runtime.Internal.Utilities;
+
+namespace Microsoft.ML.Runtime.Data
+{
+    public sealed partial class TextLoader
+    {
+        public static DataReader<IMultiStreamSource, TTupleShape> CreateReader<TTupleShape>(
```

Comment:
It would be good to add XML doc here.

Reply:
Here, and also the public members of the `Context` object.

---
In reply to: [214523219](https://github.com/dotnet/machinelearning/pull/778#discussion_r214523219) [](ancestors = 214523219)